test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/kubecfg-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
Please add a comment to each of these files with the detail you included in your little chart for this pull.  It will help future maintainers.